tests/apollo/test_skvbc_preexecution.py

```diff
@@ -82,44 +87,72 @@ async def run_concurrent_pre_execution_requests(self, skvbc, clients, num_of_req
                         nursery.start_soon(self.send_single_read, skvbc, client)
                         read_count += 1
             sent += len(clients)
+            await trio.sleep(.1)
         return read_count + write_count
 
     @unittest.skip("unstable")
     @with_trio
     @with_bft_network(start_replica_cmd)
     async def test_sequential_pre_process_requests(self, bft_network):
         """
-        Use a random client to launch one pre-process request in time and ensure that created block are as expected.
+        Use a random client to launch one pre-process request in time and ensure that created blocks are as expected.
         """
         bft_network.start_all_replicas()
         skvbc = kvbc.SimpleKVBCProtocol(bft_network)
 
-        for i in range(100):
+        for i in range(NUM_OF_SEQ_WRITES):
             client = bft_network.random_client()
             await self.send_single_write_with_pre_execution(skvbc, client)
 
+    @unittest.skip("unstable")
```

Comment:
PS: is this intentionally skipped?

Reply:
Yes. This test reproduces an issue we discussed earlier: ( f + 1) against (2f + 1) required pre-processing/client quorum inconsistency.